components/event/EventCard.js

```diff
@@ -19,20 +20,15 @@ export default function EventCard({ manage, event, usernames }) {
   const [endTime, setEndTime] = useState(event.date.end);
 
   useEffect(() => {
-    const dateTimeStyle = {
-      dateStyle: "full",
-      timeStyle: "long",
-    };
     try {
       setStartTime(
-        new Intl.DateTimeFormat("en-GB", dateTimeStyle).format(
-          new Date(event.date.start)
+        format(
+          new Date(event.date.start),
+          "eeee, d MMMM yyyy 'at' HH:mm:ss OOOO"
```

Comment:
I think the goal of the issue was to abstract the reuse of the date formatting similar to `services/utils/abbreviateNumbers.js`, I don't think need to add another library

Reply:
got it as I was a bit confused as Emma had written in the issues section that either to write helper function or to add date-fns that's why. I will do the needful changes and try to raise new modified PR tomorrow.

Reply:
I think it was an idea Emma mention, but not the goal. Ok sure, if you create a new PR please close this one, but it is best to add your changes to this PR if you can

Reply:
So, where ever dates are present I need to change it with my helper function. Right?